src/api/mod.rs

```diff
@@ -588,32 +588,52 @@ pub(crate) struct ContextInner<T: Pixel> {
   next_lookahead_output_frameno: u64,
 }
 
+/// The encoder context.
+///
+/// Contains the encoding state.
 pub struct Context<T: Pixel> {
   inner: ContextInner<T>,
   config: EncoderConfig,
   pool: rayon::ThreadPool,
   is_flushing: bool,
 }
 
+/// Status that can be returned by [`Context`] functions.
+///
+/// [`Context`]: struct.Context.html
```

Comment:
Is there a way to cross-link without specifying as HTML-only?

Reply:
Not that I know of. This is the way recommended by the API guidelines: https://rust-lang-nursery.github.io/api-guidelines/documentation.html#prose-contains-hyperlinks-to-relevant-things-c-link

Reply:
Ah, that sucks...